ThumbnailItem/ThumbnailItem.module.less

```diff
@@ -1,26 +1,75 @@
 // ThumbnailItem.module.less
 //
+@import "../styles/mixins.less";
 @import "../styles/skin.less";
 
 .thumbnailItem {
+  	align-items: center;
+
 	.thumbnail {
-		height: auto;
-		width: auto;
+	  	vertical-align: middle;
+	}
+
+	.thumbnailContainer {
+		display: flex;
```

Comment:
There may be better ways to center this, which could be more configurable by the skin, via variables, that don't affect the layout's structure.

Reply:
This is not used any more.